.chloggen/rmvalidate.yaml

```diff
@@ -0,0 +1,11 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
It is "breaking" mostly for test (may be other cases as well), where we do `factory.DefaultConfig().Validate()` which no longer compiles.